components/automate-gateway/api/iam/v2/request/teams.proto

```diff
@@ -21,7 +21,7 @@ message CreateTeamReq {
       json_schema: {
         required: ["id", "name"]
       },
-      example: { value: '{"id": "test-id", "name": "My Test Team", "projects": ["project1", "project2"]}'}
+      example: { value: '{"id": "my-team-id", "name": "My Test Team", "projects": ["project1", "project2"]}' }
```

Comment:
is this related to the issue or just a bonus fix?

Reply:
Bonus. I've been checking the payloads to eventually remove the opt-out from our UI code. 😄